### DISCUSSION TITLE
リッチメニューの画像アップロードについて

### DISCUSSION THREAD

**Discussion by hrtk7623** [harusys/newsh#45] [Q&A] [answered]
@harusys 
# ■概要
・リッチメニューの画像の作成方法は，draw.ioを門地いてVscode上で作成できることは理解
・https://developers.line.biz/ja/reference/messaging-api/#chat-room
　を参考に見ているが，具体的な方法がイマイチわからない．

# ■知りたいこと
　・ファイルはどこに保存？
　・リクエストはどこでたたく？
　・その画像をLINEにアップロードする方法を調べると，POSTMAN等のツールを用いてる例を多く見かけるが，そういったツールが必要か？

佐尾さんがアップロードした例を参考したく，簡単な手順をご連携いただきたいです

**Comment by harusys** [comment, marked as answer]
> ・ファイルはどこに保存？

GitHub に保存しています。

> ・リクエストはどこでたたく？
> ・その画像をLINEにアップロードする方法を調べると，POSTMAN等のツールを用いてる例を多く見かけるが，そういったツールが必要か？

Messaging API 経由で設定することも可能ですが、現時点ではより簡単な Web 管理画面から設定しています。
LINE Official Account Manager で「背景画像をアップロード」で GitHub に保存してある .drawio.png ファイルをアップするだけです。
参考）https://growthseed.jp/experts/sns/line-richmenu/

**Comment by hrtk7623** [comment]
なるほど，手動のアップロードだったんですね！
調べるとリクエストを投げて～というような記事ばっかりでてきたので，
勝手にそれしかないのかと思ってました．
・画像の作成/管理：draw.ioを用いてVscode上で
・アップロード：LINE Official Account Managerからアップロード
で理解しました
